src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -14,6 +14,7 @@
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Simplification;
```

Comment:
Consider swapping `using Microsoft.CodeAnalysis.Simplification;` and `using Microsoft.CodeAnalysis.Shared.Extensions;` to preserve lexical ordering.